release-note.html

```diff
@@ -23,42 +23,45 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta content="text/html; charset=iso-8859-1" http-equiv="content-type"/>
-    <title>Choreo Connect 1.0.0 - 5th January 2022</title>
+    <title>Choreo Connect 1.1.0 - 3rd April 2022</title>
 </head>
 <body>
 <div id="main-content">
     <div>
-        <h1>Choreo Connect 1.0.0 is Released.</h1>
-        <p>Choreo Connect is a cloud-native, open-source, and developer-centric API gateway. It provides first-class
-            support for Kubernetes while facilitating an array of API management quality of services (QoS), such as
-            message security rate-limiting, observability, and message mediation.</p>
-        <p>Choreo Connect 1.0.0 is compatible with WSO2 API Manager 4.0.0.</p>
-        <h1>What's new in 1.0.0</h1>
+        <h1>Choreo Connect 1.1.0 is Released.</h1>
+        <p>Choreo Connect is a cloud-native, open-source, and developer-centric API gateway. It provides first-class support for
+            Kubernetes while facilitating an array of API management quality of services (QoS), such as message
+            security rate-limiting, observability, and message mediation. It is powered by Envoy Proxy!</p>
+        <p>Choreo Connect 1.1.0 is also compatible with WSO2 API Manager 4.1.0.</p>
+        <h1>What's new in 1.1.0</h1>
         <h2>New Features</h2>
         <ul>
-            <li>Message transformations with interceptors</li>
-            <li>API key authentication</li>
-            <li>Support circuit breaking</li>
-            <li>Custom filters</li>
-            <li>Tracing with open tracing</li>
-            <li>Immutable gateway support</li>
+            <li>Mock API implementation support considering API definition.</li>
+            <li>Operational policies support with WSO2 API-M.</li>
+            <li>Request payload passing from router to enforcer.</li>
+            <li>Git integration support.</li>
+            <li>Open Policy Agent (OPA) validation support.</li>
+            <li>Support for default versioned APIs.</li>
+            <li>Analytics publishing for WebSocket APIs.</li>
```

Comment:
@pubudu538 plz check
```suggestion
            <li>Mock API implementation support considering API definition.</li>
            <li>API Policy support with WSO2 API-M.</li>
            <li>Request payload passing from router to enforcer.</li>
            <li>Git integration support.</li>
            <li>Open Policy Agent (OPA) validation.</li>
            <li>Default versioned APIs.</li>
            <li>Analytics publishing and rate limitting for WebSocket APIs.</li>
```

Reply:
+1